addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -248,7 +248,7 @@ def _get_invoice_line_item_vals(self, line, taxes_vals):
             'classified_tax_category_vals': tax_category_vals_list,
         }
 
-    def _get_document_allowance_charge_vals_list(self, invoice):
+    def _get_document_allowance_charge_vals_list(self, invoice, taxes_vals=None):
```

Comment:
@jco-odoo Isn't this breaking the stable policy ?

Reply:
It is in the grey zone as it is optional.

Reply:
I am more wondering whether taxes_vals makes a lot of sense here.  Also because you can get it again from the line data in principle.  @hsal-odoo

Reply:
It actually makes sense as you might need the taxes values in a lot of cases.